components/inputs/README.md

```diff
@@ -38,3 +39,42 @@ The `<d2l-input-text>` element is a simple wrapper around the native `<input typ
 - `step` (optional, String): for number inputs, sets the step size
 - `type` (optional, String): can be one of `text` (default), `email`, `number`, `password`, `tel`, `url`
 - `value` (optional, String): value of the input
+
+## Search Inputs
+
+For text searches use `<d2l-input-search>`, which wraps the native `<input type="search">` element.
+
+![example screenshot of search input](./screenshots/search.png?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-search.js';
+</script>
+<d2l-input-search
+  label="Search"
+  value="Apples"
+  placeholder="Search for fruit">
+</d2l-input-search>
+```
+
+**Properties:**
+
+- `disabled` (optional, Boolean): disables the input
+- `label` (required, String): accessible label for the input
```

Comment:
Should required properties be listed first?

Reply:
Hmm... that's a good question, possibly? Currently we're doing things alphabetically but that could make sense too. @svanherk @dbatiste any thoughts?

Separately, I'd like to do a pass and add in default values where it makes sense -- maybe not for booleans but for string/Number properties where we actually set defaults in the constructor.

Reply:
Ah I had thought we had been doing required first (e.g., with button https://github.com/BrightspaceUI/core/tree/master/components/button) but it could be different in different places.

Adding default values seems like a good idea.

Reply:
Haha we definitely haven't been consistent -- Dave and I went through last week with a few "consistency passes". Happy to do a pass moving required things to the top though! I'll update this one and then do the others separately.